### PR TITLE
Mo tro162 gown page

### PR DESCRIPTION
**Pull request by mdrisco4**
There needs to be a component made for the Mobile page button...or I need to make it from scratch.  I am sure some fine tuning can be done, but I think this is close enough for an initial code review.

### CODE REVIEW

**Comment by erinkelley27** [comment]
@mdrisco4 Looks like you have some merge conflicts!

**Comment by mdrisco4** [comment]
All taken care of!

File: src/components/DressListItem/DressListItem.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ const Image = styled.img`
   margin-right: auto;
   width: 100%;
 height: auto;
+<<<<<<< HEAD
```
Comment (erinkelley27) [inline src/components/DressListItem/DressListItem.js:10]: Remove lines regarding your merge conflicts here.

File: src/pages/gown.js [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+        shadow={`${"0px 4px 4px rgba(0, 0, 0, 0.25)"}`}
+        color={`${"#000000"}`}
+        desktopfontsize={`${"14px"}`}
+        desktopheight={`${"16px"}`}
+      />
+    </MobileButtonContainer>
+    <DifferentAnglesContainer>
+    <DifferentAngles src={cardData[1].image} />
+    <DifferentAngles src={cardData[2].image} />
+    <DifferentAngles src={cardData[3].image} />
```
Comment (erinkelley27) [inline src/pages/gown.js]: The dimensions on these images make them look a little bit skewed.

File: src/pages/gown.js [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+  height: 300px;
+  margin: 0 26px 0 0;
+}
+`
+
+const GownsPage = () => (
+  <Layout>
+    <NavBar />
+    <LogoContainer>
+      <Logo notag />
```
Comment (erinkelley27) [inline src/pages/gown.js:296]: The logo doesn't seem to be appearing in the browser.

File: src/pages/gown.js [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+    </DesktopButtonContainer>
+    </DesktopFlexColumn>
+    <DetailsAndFeaturesContainer>
+      <Icons name={["far", "plus-square"]} />
+      <Details>Details and Features</Details>
+    </DetailsAndFeaturesContainer>
+    </ImageAndDetailsContainer>
+    <MobileButtonContainer>
+      <PinkButton
+        label="See More From This Collection"
```
Comment (erinkelley27) [inline src/pages/gown.js]: From the figma, it looks like this button should be to Book an Appointment.

File: src/pages/gown.js [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+        <CollectionTitle>Fall 2020</CollectionTitle>
+      </DesktopCollectionTitleContainer>
+    </SearchMenuContainer>
+    <ImageAndDetailsContainer>
+    <GownImage src={cardData[0].image} />
+    <DesktopFlexColumn>
+    <Name>{cardData[0].mobileLabel}</Name>
+    <Brief>Stuff</Brief>
+    <DetailsFeatures>Details and Features</DetailsFeatures>
+    <Description>Description with a whole bunch of stuff describing the gown</Description>
```
Comment (erinkelley27) [inline src/pages/gown.js]: Try adding in some info about a dress to show what it would actually look like.

**Review by erinkelley27 — CHANGES_REQUESTED** [review]
@mdrisco4 Great start! Added some edits.

File: src/pages/gown.js (hunk 2 above)
Comment (mdrisco4) [reply src/pages/gown.js]: done

File: src/components/DressListItem/DressListItem.js (hunk 1 above)
Comment (mdrisco4) [reply src/components/DressListItem/DressListItem.js:10]: done

File: src/pages/gown.js (hunk 3 above)
Comment (mdrisco4) [reply src/pages/gown.js:296]: Logo issue is a merge issue I believe

File: src/pages/gown.js (hunk 4 above)
Comment (mdrisco4) [reply src/pages/gown.js]: still need to make button, may be able to use work that was done by Jose

File: src/pages/gown.js (hunk 5 above)
Comment (mdrisco4) [reply src/pages/gown.js]: added text from Figma mockup

**Pull request merged**